Add documentation for using CallbackFunction with Layers (#4070)
pulumi/pulumi-hugo: themes/default/content/docs/clouds/aws/guides/lambda.md

title_tag: "Using AWS Lambda & Serverless Events | Crosswalk"
title: Lambda
h1: AWS Lambda & Serverless Events

> meta_desc: Pulumi Crosswalk for AWS brings a more natural, and easier to use, way of building serverless applications using AWS Lambda.

Reviewer: ```suggestion
meta_desc: Pulumi Crosswalk for AWS offers a simple path to deploying serverless applications with AWS Lambda.
```

automatically trigger from other AWS services or call it directly from any web or mobile app.

{{% notes type="info" %}}
The features described on this page are extensions of the [AWS Classic provider](/registry/packages/aws/) and are only available in JavaScript and TypeScript.
{{% /notes %}}

## Overview

> Pulumi Crosswalk for AWS brings a more natural, and easier to use, way of building serverless applications using

Reviewer 2: ```suggestion
Pulumi Crosswalk for AWS brings a more natural, and easier to use way of building serverless applications using
```

> AWS Lambda. Pulumi lets you express Lambda functions using real code, and handles packaging, versioning, and

Author: Ah yeah, good spot; I read right past that. (It's part of the existing doc.) I'll adjust.

configuration of the associated AWS resources for you. This lets you focus on your application logic without needing to
worry about boilerplate, and with confidence that the resulting infrastructure automatically uses AWS best practices.

With Pulumi Crosswalk for AWS, event sources are available on all native resource types, including AWS S3, SQS,
DynamoDB, CloudWatch, Kinesis, and more, in addition to full support for [API Gateway](/docs/clouds/aws/guides/api-gateway/).

> This improves discoverability of event sources in addition to adding strong typing to the event handler

Reviewer 2: ```suggestion
This improves the discoverability of event sources and adds strong typing to the event handler
```
